client/src/utils/mountVueComponent.js

```diff
@@ -22,17 +22,28 @@ Vue.use(vueRxShortcutPlugin);
 // font-awesome svg icon registration/loading
 Vue.use(iconPlugin);
 
+function getOrCreatePinia() {
+    // We sometimes use this utility mounting function in a context where there
+    // is no existing vue application or pinia store (e.g. individual charts
+    // displayed in an iframe).
+    // To support both use cases, we will a new pinia store and attach it to the
```

Comment:
```suggestion
    // To support both use cases, we will create a new pinia store and attach it to the
```
although I also like the concept of `will`ing something into existence 😆 .